flopy/utils/datafile.py

```diff
@@ -221,9 +221,7 @@ def __init__(self, filename: Union[str, os.PathLike], precision, verbose, **kwar
         if self.mg is None:
             self.mg = StructuredGrid(
                 delc=np.ones((self.nrow,)),
-                delr=np.ones(
-                    self.ncol,
-                ),
+                delr=np.ones(self.ncol),
```

Comment:
This is where the code was probably originally `np.ones(self.ncol,)` in attempt to specify a tuple (which would be done with `np.ones((self.ncol,))`). Fortunately, this is harmless, as numpy's shape can be an integer to mean the same as a tuple of one integer. No change is recommended here.